docs/manual/java/guide/build/EclipseSbt.md

```diff
@@ -8,30 +8,30 @@ Follow these steps to integrate your project with Eclipse:
 1. [Create an External Tool Configuration](#Create-an-External-Tool-Configuration)
 
 # Import the project
-  
+
 1. In a console, `cd` to the top-level folder of your existing sbt project, and enter `sbt eclipse` to generate the Eclipse project files for all projects in your build.
     The sbt plugin creates `.project` and `.classpath` files for the subprojects. The last few lines of output confirm success:
-    
-    ```
-    ...
-    [info] Successfully created Eclipse project files for project(s):
-    [info] hello-impl
-    [info] hello-api
-    [info] hello-stream-impl
-    [info] hello-stream-api
-    [info] lagom-internal-meta-project-service-locator
-    [info] lagom-internal-meta-project-cassandra
-    
-    ```
+
+```
+...
+[info] Successfully created Eclipse project files for project(s):
+[info] hello-impl
+[info] hello-api
+[info] hello-stream-impl
+[info] hello-stream-api
+[info] lagom-internal-meta-project-service-locator
+[info] lagom-internal-meta-project-cassandra
+
+```
```

Comment:
What was the reason for this change? IMHO, the previous formatting was better.

Contrast this: https://github.com/ignasi35/lagom/blob/9ace3541cb36bc1f243588251596dd170d7859d1/docs/manual/java/guide/build/EclipseSbt.md

<img width="949" alt="screen-shot-new" src="https://cloud.githubusercontent.com/assets/44385/23386653/c9a11ebc-fdab-11e6-82bd-d5ea6dc41f85.png">


with the previous version: https://github.com/ignasi35/lagom/blob/9ace3541cb36bc1f243588251596dd170d7859d1^/docs/manual/java/guide/build/EclipseSbt.md

<img width="961" alt="screen-shot-prev" src="https://cloud.githubusercontent.com/assets/44385/23386659/d7d896a4-fdab-11e6-8e62-74593d9a9985.png">

The new version creates two separate numbered lists instead of continuing with step 2 after the output snippet.